src/components/sbb-link/sbb-link.tsx

```diff
@@ -99,6 +108,20 @@ export class SbbLink implements AccessibilityProperties, ComponentInterface {
    */
   @State() private _isStatic = false;
 
+  /** Id which is sent in the click event payload */
+  @Prop() public eventId?: string;
+
+  /**
+   * Emits whenever the native button click event triggers.
+   * TODO: similar to the one in sbb-button. To be fixed together.
```

Comment:
what should be fixed?

Reply:
Hello Jeri, 
I think it's about removing the 'any' type; Lukas add this comment in the button event
https://github.com/lyne-design-system/lyne-components/blob/master/src/components/sbb-button/sbb-button.tsx#L58